app/helpers/application_helper.rb

```diff
@@ -1,7 +1,21 @@
 require 'csv'
 require 'json'
+require 'open3'
 
 module ApplicationHelper
+  def sanitize(user_input_text)
+    # accept alpanumeric characters only
```

Comment:
typo: alpanumeric -> alphanumeric

Maybe change to "accept alphanumeric characters and symbols `_` and `-`"